language-server/dm/documentManager.ml

```diff
@@ -739,6 +739,48 @@ let hover st pos =
   let lconstr = Procq.Constr.lconstr
 [%%endif]
 
+
+[%%if coq = "8.18" || coq = "8.19" || coq = "8.20"]
+let jump_to_definition _  _ = None
+[%%else]
+let jump_to_definition st pos =
+  let raw_doc = Document.raw_document st.document in
+  let loc = RawDocument.loc_of_position raw_doc pos in
+  let opattern = RawDocument.word_at_position raw_doc pos in
+  match opattern with
+  | None -> log "jumpToDef: no word found at cursor"; None
+  | Some pattern ->
+    log ("jumpToDef: found word at cursor: \"" ^ pattern ^ "\"");
+    try
+    let qid = parse_entry st loc (Procq.Prim.qualid) pattern in
+      let ref = Nametab.locate_extended qid in
```

Comment:
I guess this and maybe the locate_absolute_library are where Not_found can come from

Reply:
Yeah I just enclosed it in a big ```try [...] with``` and it no longer breaks the server so that's okay